src/engraving/dom/timesig.h

```diff
@@ -116,6 +116,11 @@ class TimeSig final : public EngravingItem
 
     bool isLocal() const { return m_stretch != Fraction(1, 1); }
 
+    PointF staffOffset() const override;
+
+    bool isCourtesy() const { return m_isCourtesy; }
```

Comment:
I would probably do this as a funcion, rather than storing it as a state variable. Of course, doing it as a state variable makes it super fast to retrieve... but it also adds a state variable, which needs to be set at the right moment, reset at the right moment, and generally kept up to date. To determine whether a timeSig or keySig is a courtesy, wouldn't it be enough to do something like
```
bool isCourtesy() {
    return segment()->tick() == segment()->measure()->system()->endTick();
}
```